### PR TITLE
#8: Initialize gitignore file

### LINKED ISSUE #8: Initialize gitignore file

### PR DESCRIPTION
**Pull request by rayestrada**
Closes #8

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Ignore files generated by common IDEs
+.idea
+.vscode
+
+# Ignore OS generated files
+.DS_Store*
+ehthumbs.db
+Thumbs.db
+._*
```
Comment (mejiaj) [inline .gitignore:9]: Should we include `node_modules`?

```suggestion
._*

# Ignore Node generated files.
node_modules

```

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Ignore files generated by common IDEs
+.idea
+.vscode
+
+# Ignore OS generated files
```
Comment (mejiaj) [inline .gitignore]:
```suggestion
# Ignore OS generated files.
```

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Ignore files generated by common IDEs
```
Comment (mejiaj) [inline .gitignore]:
```suggestion
# Ignore files generated by common IDEs.
```

**Review by mejiaj — COMMENTED** [review]
Looks good. I've added optional suggestions.

File: .gitignore (hunk 1 above)
Comment (rayestrada) [reply .gitignore:9]: This directory shouldn't have node_modules in it. It is a base theme and all node packages would be installed the in generated child theme outside of this code. I think we are okay to leave it out unless there becomes a use case.

**Comment by rayestrada** [comment]
Added periods in comments

**Pull request merged**